src/frontend/src/common/analytics/eventTypes.ts

```diff
@@ -129,9 +129,24 @@ export type AnalyticsTreeCreationNextstrain = {
   // Type of tree being created
   tree_type: string;
   // Location of samples used for tree creation. A null value indicates that
-  // the users group info was not successfully fetched before they created
+  // the user's group info was not successfully fetched before they created
   // a tree. This generally shouldn't happen.
   location_id: number | null;
+  // Location of the user's group. This is provided for comparison with the
+  // location_id above. A null value indicates that the user's
+  // group was not successfully fetched before they created a tree. This
+  // generally shouldn't happen.
+  group_location_id: number | null;
+  // Lineages selected for tree creation.
+  selected_lineages: string;
+  // Time range filter for tree creation. A null value indicates that
+  // the user did not change the default value and there is
+  // no start date set.
+  start_date: FormattedDateType | null;
```

Comment:
`FormattedDateType` is generally a YYYY-MM-DD string, right? It looks like where the `FormattedDateType` is defined, it doesn't actually say what the format of the string is, but I think it must be YYYY-MM-DD. That should probably be documented there, although I know you didn't make that type.

Also... I'm not actually sure what will happen when this lands in the DB. It's a new field, so the DB Warehouse service will autogenerate a migration to add this new column, but I don't know if it will make a string or a Date out of it in the DB. ... guess we'll find out? It's probably not a big deal either way, I assume this isn't a critical field, and it should be possible to cast the string to a date during analytics if it lands as a date.

Reply:
I assumed it would land as a string, but I don't have any basis for this assumption.  I'll check with Amy - I assume she's the only person who might care.  I can also add to the comment to describe the format.